Gemfile.lock

```diff
@@ -61,11 +55,14 @@ GEM
     docopt (0.5.0)
     domain_name (0.5.24)
       unf (>= 0.0.5, < 1.0.0)
-    dor-services (4.21.4)
-      active-fedora (~> 5.7.1)
-      activesupport (~> 3.2, >= 3.2.18)
+    dor-rights-auth (1.0.2)
+      nokogiri
+    dor-services (5.2.0)
```

Comment:
It could be (and would be if updated again).  5.x is enough for me to accept that the repo is compatible though.